packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+// eslint-disable-next-line no-restricted-imports
+import * as Ariakit from '@ariakit/react';
+
+/**
+ * Internal dependencies
+ */
+import { COLORS } from '../utils';
+import { space } from '../utils/space';
+import type { CustomSelectProps } from './types';
+
+export const CustomSelectLabel = styled( Ariakit.SelectLabel )`
+	font-size: 11px;
```

Comment:
Should we try using a font size consistent with the rest of the design system? For example `fontSizeSmall` which I think should result in around 12 px?

```suggestion
	font-size: ${ CONFIG.fontSize };
```

given that we import `CONFIG` as follows:

`import { CONFIG } from '../../utils';`

Reply:
I guess for now Brooke was replicating the styles of the legacy `CustomSelectControl` component, which internally uses a `Button` component, which [has an `11px` font size when `size="small"`](https://github.com/WordPress/gutenberg/blob/030e2cf6a8307fba06c40be872e4de7c879734f0/packages/components/src/button/style.scss#L274).

But this is definitely something that we should keep in mind when re-evaluating this component's design — we should either bring 11px to the list of available font sizes, or use a font size that is already defined in the config.

Reply:
Sounds good, thanks for confirming!